doc/devel/MEP/MEP12.rst

```diff
@@ -87,6 +87,22 @@ the gallery would remain in those directories until they are cleaned
 up. After clean-up, they would be moved to one of the new gallery
 sections described above. "Clean-up" should involve:
 
+* `sphinx-gallery docstrings <http://sphinx-gallery.readthedocs.io/>`_: a
+  title and a description formatted as follows::
```

Comment:
I think this should probably mention that it's the docstring for the entire file (right?), not some function within an example.